pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -14,6 +14,15 @@
     "sha256": "1wdrjpd3l0xadsa3lqhsc9c57g8x2qkwb76q824sk8za1a7lapii",
     "version": "1.5.0-patched"
   },
+  "aiven": {
+    "owner": "aiven",
+    "provider-source-address": "registry.terraform.io/aiven/aiven",
```

Comment:
When is this required, and when not? Do we have some docs about this?

Noticed I didn't set this for https://github.com/NixOS/nixpkgs/pull/125665…

Reply:
Thanks! Added to https://github.com/NixOS/nixpkgs/pull/126203#discussion_r647408473